src/Stack/Dot.hs

```diff
@@ -220,9 +220,9 @@ pruneGraph :: (F.Foldable f, F.Foldable g, Eq a)
            -> Map PackageName (Set PackageName, a)
 pruneGraph dontPrune names =
   pruneUnreachable dontPrune . Map.mapMaybeWithKey (\pkg (pkgDeps,x) ->
-    if show pkg `F.elem` names
+    if packageNameString pkg `F.elem` names
```

Comment:
What about going other way around? I.e. compare package names and not strings? Not a large difference but adds a bit of type safety

Reply:
Good idea!